src/concepts/glean/accessing_glean_data.md

```diff
@@ -1,13 +1,34 @@
-# Accessing Glean data
+# Accessing Glean Data
 
 This document describes how to access Glean data from an SQL query, such as in [Redash](https://sql.telemetry.mozilla.org).
 
-## Selecting the correct table
+## Using the Glean Dictionary
+
+You can see a full index of how each Glean applications maps to structures in BigQuery using the [Glean Dictionary](https://dictionary.protosaur.dev/).
+
+Select the application you're interested in from the home page, then browse through the metrics and pings. You will see the appropriate table and column
+names to use when running a query. In general, you can use these types of column names directly. For example, to get a count of distinct clients in the [Fenix baseline ping](https://dictionary.protosaur.dev/apps/fenix/pings/baseline), run this:
+
+```sql
```

Comment:
I don't think example would have helped me identify my use case. For me, I would have followed the instructions and:
- Selected fenix
- Search for `top_sites_count` & select it

But I would not have known what to do with the info on the next page. The key thing I was missing was that this:

![image](https://user-images.githubusercontent.com/759372/110861322-d4065700-8272-11eb-888d-d7690fb1b92f.png)


means that I can actually write a SELECT query on it, e.g.
```sql
SELECT
  metrics.counter.metrics_top_sites_count
FROM
  org_mozilla_fenix.metrics
WHERE ...
```

I think for your example, it might be clearer if we do one or some of:
- we explicitly mention what content on that page can be used in BigQuery
- improve the page itself to make it clearer how those can be used in queries (with an example?)
- (least preferred) change the link we send to `client_info` rather than the baseline ping docs – I feel the pattern matching is easier there (because what I'm missing is how to extract fields from a top-level select statement item)

Reply:
This is really helpful feedback, thanks @mcomella! I'll give this a bit more thought, it didn't occur to me that the interface in the glean dictionary didn't make this obvious.